common/trainer/default_trainer.py

```diff
@@ -130,7 +131,7 @@ def prepare(self, model, tokenizer, dataset, optimizer, accelerator):
     def train(self):
         num_train_epochs = self.config.get("num_train_epochs", 1)
         checkpoint = self.config.get("checkpoint")
-        log_step = self.config.get("log_step", 1)
+        logging_steps = self.config.get("logging_steps")
```

Comment:
Please add default value `1` because of ui.

Reply:
OK. I've set the default to 10 in the configuration file. Fine for UI?

Reply:
Got it. It‘s because now start_ui.py does not use finetune.yaml, so the 'logging_steps' will be none. Or I modify start_ui.py later to get the configuration from finetune.yaml.

Reply:
I understood it. Sorry I didn't say clearly. I mean I will update this to `logging_steps = self.config.get("logging_steps", 10)` here using 10 instead of 1 as default.

Reply:
OK, but progress bar needs the values of `report({})` to be updated every step, otherwise the status of progress bar will only be updated after 10 steps.

Reply:
I see. Ok, will set it to 1 here.